Add eZPlatform index subtree cron and add cron to frequent jobs

harmstyler opened the pull request:
The cronjob is mostly the same as the eZ Find version (ezfindexsubtree.php), with some very small modifications. I also added it to the frequent cronjobs, which eZ Find did as well.

I wasn't sure I needed to keep the license comment at the top, since I did change some things in the file. I can always add it back in.

harmstyler committed:
Add eZ Platformm index subtree cron and add cron to frequent jobs

emodric commented on cronjobs/ezplatformindexsubtree.php:
```diff
@@ -0,0 +1,72 @@
+}
+
+// check that solr is enabled and used
+$eZSolr = eZSearch::getEngine();
```
Variable name should be changed, togethercwith comment above not to reference solr :)

harmstyler replied:
`$eZSearch` work?

emodric replied:
$searchEngine ?

emodric commented on settings/cronjob.ini.append.php:
```diff
@@ -0,0 +1,12 @@
+[CronjobPart-ezplatformindexsubtree]
+Scripts[]=ezplatformindexsubtree.php
+
+*/
```
New line at EOF is missing here.

harmstyler replied:
Fixed with 3f7e799

emodric commented on settings/cronjob.ini.append.php:
```diff
@@ -0,0 +1,12 @@
+[CronjobSettings]
+ExtensionDirectories[]=ezplatformsearch
+
+[CronjobPart-frequent]
```
Is it a good idea to add it to frequent?

I'd rather leave more control to users if this should appear in frequent part or not.

harmstyler replied:
eZ Find adds it to frequent, so I figured I would as well since it serves the same purpose. If its not added to frequent the README should probably be updated to make sure users run the cron.

emodric replied:
And I always removed it from frequent part :D

Adding a note to README is okay, ofcourse.

harmstyler committed:
Adds missing line at EOF

harmstyler committed:
remove ezplatformindexsubtree.php from frequent and update README

harmstyler committed:
Update variable name and spelling error fix for README

Updated the search engine variable in the cron to a more meaningful name
Fixed a spelling error on the README

harmstyler commented:
I removed the cron from the frequent crons list and updated the variable name for the search engine used in the cronjob.

emodric commented:
Thanks @harmstyler !
